.changeset/unlucky-stingrays-appear.md

```diff
@@ -0,0 +1,96 @@
+---
+'@commercetools-uikit/design-system': patch
```

Comment:
I used `patch` as suggested by @emmenko to verify what will be the resulting `changesets` PR